library/src/main/java/com/novoda/downloadmanager/lib/Request.java

```diff
@@ -234,6 +235,17 @@ public Request alwaysAttemptResume() {
     }
 
     /**
+     * Automatically pause the download when reaching the end of the file instead of writing the last bytes to disk.
+     * This allows for a resume of the download against an updated tar file.
+     * If used this also sets {@code alwaysAttemptResume()}
+     */
+    public Request allowTarUpdates() {
+        allowTarUpdates = true;
+        alwaysAttemptResume();
```

Comment:
hmmm, OK, I'd prefer if this is the client's responsibility I think, it's hidden knowledge. But definitely mention that this method is perhaps required in the javadoc?